docs/projects/modules/jam/pages/suggest.adoc

```diff
@@ -1,45 +1,118 @@
-= Suggest a presentation
+= Suggest a presentation 
```

Comment:
```suggestion
= Suggest a presentation
```

Reply:
Thanks @pateldivyesh1323 for the PR! Everything else looks good except for one minor change requested.